Formula/latino.rb

```diff
@@ -0,0 +1,33 @@
+class Latino < Formula
+  desc "Lenguaje de programación de código abierto para latinos y de habla hispana"
+  homepage "https://www.lenguajelatino.org/"
+  license "MIT"
+  stable do
+    url "https://github.com/lenguaje-latino/Latino/archive/v1.3.0.tar.gz"
+    sha256 "28118a047265394ad9a5fcb2c76895d08af4b43a5886f52afce98a0f8cdc86fe"
+    depends_on "cmake" => :build
+    depends_on "make" => :build
+  end
+  head do
+    url "https://github.com/lenguaje-latino/Latino.git"
+    depends_on "cmake" => :build
+    depends_on "make" => :build
+  end
+  fails_with :clang do
+    build 600
+    cause "multiple configure and compile errors"
+  end
+  fails_with :gcc do
+    build 600
+    cause "multiple configure and compile errors"
+  end
+  def install
+    mv "logo/Latino-logo.png", "logo/desktop.png"
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "install"
+  end
+  test do
+    system bin/"latino", "-v"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Yes, here is the new and improved test 

```
  test do
    (testpath/"test1.lat").write "poner('hola mundo')"
    (testpath/"test2.lat").write "i=0\nrepetir\n\tponer(i)\n\ti++\nhasta(i>=10)"
    output = shell_output("#{bin}/latino test1.lat")
    assert_equal "hola mundo", output.chomp
    output2 = shell_output("#{bin}/latino test2.lat")
    assert_equal "0\n1\n2\n3\n4\n5\n6\n7\n8\n9", output2.chomp
  end
```

This creates 2 small `latino` programs and runs them and then compares their outputs with the expected output